doc/content/getting-started/installation/configuration/docker-configuration/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Docker Configuration"
+description: ""
+weight: 2
+---
+
+In this section, configuring Docker is explained through an example of `docker-compose.yml` file.
+
+<!--more-->
+
+Docker runs an instance of {{% tts %}}, as well as an SQL database and a Redis database, which {{% tts %}} depends on to store data.
+
+In `docker-compose.yml` file, Docker is configured to run three services:
+
+- An SQL database (CockroachDB and PostgreSQL are supported)
+- Redis
+- {{% tts %}}
+ 
+### SQL Database
+
+To configure an SQL database, a single instance of [CockroachDB](https://www.cockroachlabs.com/) is used in this guide. Note that the `volumes` need to be set up correctly so that the database is persisted on your server's disk.
+
+The simplest configuration for CockroachDB looks like this (excerpted from the example `docker-compose.yml`):
+
+{{< highlight yaml "linenos=table,linenostart=5" >}}
+{{< readfile path="/content/getting-started/installation/configuration/docker-compose-enterprise.yml" from=5 to=14 >}}
+{{< /highlight >}}
+
+>**Note:** It is also possible (and even preferred) to use a managed SQL database. In this case, you will need to update the [`is.database-uri` configuration option]({{< ref "/reference/configuration/identity-server#database-options" >}}) to point to the address of the managed database.
```

Comment:
This is not correct when running with Docker, the `TTN_LW_IS_DATABASE_URI` is always set and will override the `is.database-uri` in the config file.

For Docker, the database URI is always configured with the `TTN_LW_IS_DATABASE_URI` environment variable defined in the `docker-compose.yml` file.